src/App.tsx

```diff
@@ -52,6 +52,7 @@ function App(isSignedIn: AppProps) {
       <AuthenticatedRoute path="/confirm-payment" component={FinancialSummaryPayment} />
 
       <AuthenticatedRoute path="/inventory" component={Inventory} />
+      <AuthenticatedRoute path="/inventory/create" component={AddInventory} />
```

Comment:
I think the error you were talking about when navigating to `/inventory/create` is caused by the lines here. The path `/inventory/create` triggers both of these I think since `/inventory/create` is a suffix of `/inventory`. To fix, we can put `exact` in both routes so `/inventory` won't trigger on `/inventory/create`